scripts/webpack.js

```diff
@@ -44,8 +44,21 @@ function rules() {
 
 function plugins() {
   return {
-    ignoreEsprima: () => new webpack.IgnorePlugin(/^esprima$/, /js-yaml/),
-    ignoreMomentOptionalDeps: () => new webpack.IgnorePlugin(/^\.\/locale$/, /moment$/),
+    ignoreEsprima: () =>
+      new webpack.IgnorePlugin({
+        resourceRegExp: /^esprima$/,
+        contextRegExp: /js-yaml/,
+      }),
+    ignoreMomentOptionalDeps: () =>
+      new webpack.IgnorePlugin({
+        resourceRegExp: /^\.\/locale$/,
+        contextRegExp: /moment$/,
+      }),
```

Comment:
The syntax for [IgnorePlugin changed](https://webpack.js.org/migrate/5/#clean-up-configuration) ([new documentation](https://webpack.js.org/plugins/ignore-plugin/#using-regular-expressions)): 

> If you are using `IgnorePlugin` with a regular expression as argument, it takes an `options` object now: `new IgnorePlugin({ resourceRegExp: /regExp/ })`.